includes/payment-methods/class-wc-gateway-stripe-boleto.php

```diff
@@ -145,7 +145,7 @@ public function payment_fields() {
 		<input id="stripe_boleto_tax_id" name="stripe_boleto_tax_id" type="text"><br><br>
 		<div class="stripe-source-errors" role="alert"></div>
 
-		<div id="stripe-boleto-payment-data"><?php echo wpautop( esc_html( $description ) ); ?></div>
+		<div id="stripe-boleto-payment-data"><?php echo wp_kses_post( wpautop( $description ) ); ?></div>
```

Comment:
Anywhere that wasn't already previously using `wp_kses_post` (which I can flag in this PR), we may be better off using `wp_kses` and passing in the paragraph tag as the only allowable HTML. Should be slightly faster in regards to performance but also ensures we're only allowing paragraphs, which should match existing functionality